lib/registration.pm

```diff
@@ -731,8 +731,13 @@ sub yast_scc_registration {
     # for leap to sle migration, we need to install yast2-registration
     # before running yast2 registration module.
     my $client_module = 'scc';
+    # For leap to SLES migratoin, we need install yast2-registration and enable
+    # start rollback.service before registration.
     if (is_leap_migration) {
         zypper_call('in yast2-registration');
+        zypper_call('in rollback-helper');
```

Comment:
I suggest to create a function to do this to avoid code redundancy.

Reply:
changed to use the common_service_start function.